open system shell  short key

kr-g opened the pull request:
this is regarding

https://github.com/thonny/thonny/issues/2379

aivarannamaa commented:
See my reply under https://github.com/thonny/thonny/issues/2379

Also, I wouldn't accept this PR even if I wanted to include this change, because it has unrelated changes next to it. Each feature should be submitted from a separate branch.

kr-g commented:
yes true, i already cleaned up my repo yesterday evening because somehow i updated the branch with github some long time ago from the master (what was wrong, since that makes of course no sense on a branch, but github offered that button.. and ... i always press buttons...) ... or i dont know why this happend. mostlikely its also already since the begining where i worked directly on the master of my repo...
